release.py

```diff
@@ -358,38 +357,37 @@ def pypi_release(opts, pargs):
     version = pargs[1]
 
     db_warning = "Warning: No new distribution build. This occurs when there are no new changes to the source code since the previous release. Please check for any untracked changes and update your package changelog/release-history to reflect the newest version."
-
-    # Only upload source
-    if opts.build_no_wheel:
-        call("python -m build --sdist", release_dir)
+    build_wheel = not opts.no_wheel
+    # Build wheel and source
+    if build_wheel:
+        call("python -m build", release_dir) 
         # Upload using twine
         no_tar = True
+        no_whl = True
         for file in list((release_dir / "dist").iterdir()):
             if re.search(f".*{version}.*.tar.gz", file.name):
                 no_tar = False
+            if re.search(f".*{version}.*.whl", file.name):
+                no_whl = False
         if no_tar:
             call(f"echo \"{db_warning}\"", release_dir)
+        elif no_whl:
+            call(f"echo \"Warning: No wheel found.\"", release_dir)
         else:
-            call(f"twine upload dist/*{version}*.tar.gz", release_dir)
+            call(f"twine upload dist/*{version}*.tar.gz dist/*{version}*.whl", release_dir)
 
-    # Build wheel and source (Default behavior)
+    # Only upload source
     else:
```

Comment:
its redundant, but I wonder if it is more readable if we put `elif not build_wheel`.  I am not sure about this one.....

Reply:
Let's merge it but make an issue

Reply:
Not super high priority